framework/PageFramework.css

```diff
@@ -42,16 +40,3 @@
 :root:where(.pf-v5-theme-dark) .bg-lighten {
   background-image: linear-gradient(#ffffff08, #ffffff08);
 }
-
```

Comment:
I would like to leave there until we figure out if pf5 needs them for the toolbar to wrap properly.